test/e2e/util.go

```diff
@@ -481,9 +482,14 @@ func kubectlCmd(args ...string) *exec.Cmd {
 	}
 	kubectlArgs := append(defaultArgs, args...)
 
-	//TODO: the "kubectl" path string might be worth externalizing into an (optional) ginko arg.
-	cmd := exec.Command(filepath.Join(testContext.RepoRoot, "cluster/kubectl.sh"), kubectlArgs...)
-	Logf("Running '%s %s'", cmd.Path, strings.Join(cmd.Args, " "))
+	//Allow users to specify KubectlPath at commandline... so that we can test kubectl (the binary)
+	//Or so that, in the dev mode, we can test the kubectl shell script.
+	cmdToRun := testContext.KubectlPath
```

Comment:
sorry for so many nits, but I don't think having cmdToRun as a separate variable really makes sense. just fold this into exec.Command below.